3.2.2 doesn't include project files

v3.2.2 doesn't include project files
![image](https://user-images.githubusercontent.com/43852295/177404224-bbb02abe-e4c6-4752-b96b-7f2a8fbdfa2b.png)

These are the only 3 files:
![image](https://user-images.githubusercontent.com/43852295/177404351-aef7c780-b7e5-44f9-a228-c1d0e9167f72.png)

Transition from version `3.2.1` to `3.2.2` caused our CI pipeline to fail.
Errors like these appear in the eslint log:

```
   1:1   error    Resolve error: typescript with invalid interface loaded as resolver  import/namespace
   1:1   error    Resolve error: typescript with invalid interface loaded as resolver  import/no-unresolved
   1:1   error    Resolve error: typescript with invalid interface loaded as resolver  import/default
   1:1   warning  Resolve error: typescript with invalid interface loaded as resolver  import/no-duplicates
   1:1   error    Resolve error: typescript with invalid interface loaded as resolver  import/no-extraneous-dependencies
   1:1   error    Resolve error: typescript with invalid interface loaded as resolver  import/order
   1:1   error    Resolve error: typescript with invalid interface loaded as resolver  import/no-self-import
   1:1   error    Resolve error: typescript with invalid interface loaded as resolver  import/no-cycle
   1:1   error    Resolve error: typescript with invalid interface loaded as resolver  import/no-relative-packages
   1:1   warning  Resolve error: typescript with invalid interface loaded as resolver  import/no-named-as-default
   1:1   warning  Resolve error: typescript with invalid interface loaded as resolver  import/no-named-as-default-member
   1:1   error    Resolve error: typescript with invalid interface loaded as resolver  import/extensions
   1:1   error    Resolve error: typescript with invalid interface loaded as resolver  import/no-useless-path-segments
   1:1   error    Resolve error: typescript with invalid interface loaded as resolver  import/named
```

--- a/.changeset/eighty-plums-battle.md
+++ b/.changeset/eighty-plums-battle.md
@@ -1,0 +1,5 @@
+---
+"eslint-import-resolver-typescript": patch
+---
+
+fix: run prerelease manually for yarn v3

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -34,7 +34,7 @@ jobs:
         with:
           version: yarn changeset version
           # This expects you to have a script called release which does a build for your packages and calls changeset publish
-          publish: yarn release
+          publish: yarn prerelease && yarn release
           commit: 'chore: release eslint-import-resolver-typescript'
           title: 'chore: release eslint-import-resolver-typescript'
         env:

